lib/react_on_rails/server_rendering_pool/ruby_embedded_java_script.rb

```
@@ -230,7 +230,9 @@ def parse_result_and_replay_console_messages(result_string, render_options)
           result = nil
           begin
             result = JSON.parse(result_string)
+            binding.pry
```

rubocop

[warning] Remove debugger entry point `binding.pry`.

(Lint/Debugger)